tubesync/sync/models.py

```diff
@@ -1287,6 +1287,19 @@ def playlist_title(self):
         field = self.get_metadata_field('playlist_title')
         return self.loaded_metadata.get(field, '')
 
+    @property
+    def selected_path(self):
+        if self.downloaded and self.media_file:
+            filename = self.media_file.path
+            for prefix in self.source.directory_path.as_posix().split('/'):
+                if prefix == '':
+                    continue
+                filename = filename.removeprefix(f'/{prefix}')
+            filename = filename.removeprefix('/')
+        else:
+            filename = self.filename
+        return os.path.splitext(filename)[0]
```

Comment:
The base name that was removed here causes the duplication of subdirectories when the full path is constructed.